src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -367,9 +369,12 @@ const UserEnrollmentForm = ({
                 default: 'Confirm registration',
                 pending: 'Confirming registration...',
                 complete: 'Registration confirmed',
-                error: 'Try again',
+                error: formSubmissionError && formSubmissionError?.message?.includes('duplicate order')
```

Comment:
[curious] Is `isDuplicateOrder` a relevant helper function to use here (and on line 377 below) as well?

Reply:
Yep, you're correct!